### PR TITLE
[Improve] Unified Mapper

### PR DESCRIPTION
**Pull request by wuchunfu**
## What changes were proposed in this pull request

Issue Number: #1771 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

**Comment by wuchunfu** [comment]
@wolfboys PTAL, thanks

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java [hunk 1]
```diff
@@ -133,7 +133,9 @@ public void updateMenu(Menu menu) throws Exception {
     public void deleteMenus(String[] menuIds) throws Exception {
         for (String menuId : menuIds) {
             // Find users associated with these menus/buttons
-            this.baseMapper.deleteRoleMenuByMenuId(Long.parseLong(menuId));
+            LambdaQueryWrapper<Menu> queryWrapper = new LambdaQueryWrapper<>();
+            queryWrapper.eq(StringUtils.isNotEmpty(menuId), Menu::getMenuId, Long.parseLong(menuId));
+            this.remove(queryWrapper);
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java]: There is a bug here,It should be delete record in table `t_role_menu`, not  `t_menu`

Comment (wuchunfu) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java]: I was careless, thanks for the reminder, I want to fix it

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java [hunk 2]
```diff
@@ -133,7 +137,9 @@ public void updateMenu(Menu menu) throws Exception {
     public void deleteMenus(String[] menuIds) throws Exception {
         for (String menuId : menuIds) {
             // Find users associated with these menus/buttons
-            this.baseMapper.deleteRoleMenuByMenuId(Long.parseLong(menuId));
+            LambdaQueryWrapper<RoleMenu> queryWrapper = new LambdaQueryWrapper<>();
+            queryWrapper.eq(StringUtils.isNotEmpty(menuId), RoleMenu::getMenuId, Long.parseLong(menuId));
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java]: Code could be more concise: ` queryWrapper.eq(RoleMenu::getMenuId, menuId);`

Comment (wuchunfu) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java]: Done

**Comment by wolfboys** [comment]
hi wuchunfu:
thanks for your contribution, I see that you changed the parameterType and resultType to the full class name, good job, but many mapper.xml under the core module have not been changed, so can you modify them all together?

**Comment by wuchunfu** [comment]
> hi wuchunfu: thanks for your contribution, I see that you changed the parameterType and resultType to the full class name, good job, but many mapper.xml under the core module have not been changed, so can you modify them all together?

Yes, very willing to do this, but I plan to do this with another PR

**Comment by wolfboys** [comment]
> > hi wuchunfu: thanks for your contribution, I see that you changed the parameterType and resultType to the full class name, good job, but many mapper.xml under the core module have not been changed, so can you modify them all together?
> 
> Yes, very willing to do this, but I plan to do this with another PR

ok, looking forward to your next PR

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java [hunk 3]
```diff
@@ -133,7 +137,9 @@ public void updateMenu(Menu menu) throws Exception {
     public void deleteMenus(String[] menuIds) throws Exception {
         for (String menuId : menuIds) {
             // Find users associated with these menus/buttons
-            this.baseMapper.deleteRoleMenuByMenuId(Long.parseLong(menuId));
+            LambdaQueryWrapper<RoleMenu> queryWrapper = new LambdaQueryWrapper<>();
+            queryWrapper.eq(RoleMenu::getMenuId, Long.parseLong(menuId));
+            this.roleMenuMapper.delete(queryWrapper);
             // Recursively delete these menus/buttons
             this.baseMapper.deleteById(menuId);
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java]: I recommend to use this method: `this.removeById(menuId);`

Comment (wuchunfu) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java]: Done

**Review by wolfboys — APPROVED** [review]
LGTM

**Pull request merged**